keyutil/key.go

```diff
@@ -32,6 +33,24 @@ var (
 	MinRSAKeyBytes = 256
 )
 
+type atomicBool int32
```

Comment:
I'd have gone with a simple variable (accessed atomically) but what you did offers a straighter line to a potential upgrade for 1.19.